### ISSUE TITLE
CI mysqltest `infoschema` failed

### ISSUE THREAD

**Issue by aytrack** [pingcap/tidb#29832]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
in ci https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_common_test/detail/tidb_ghpr_common_test/7359/pipeline/86
```
run test [infoschema] err: sql:SELECT * from INFORMATION_SCHEMA.SCHEMATA;: failed to run query
"SELECT * from INFORMATION_SCHEMA.SCHEMATA;"
 around line 15,
we need(526):
SELECT * from INFORMATION_SCHEMA.SCHEMATA;
CATALOG_NAME    SCHEMA_NAME     DEFAULT_CHARACTER_SET_NAME      DEFAULT_COLLATION_NAME  SQL_PATH
def     d       utf8mb4 utf8mb4_bin     NULL
def     INFORMATION_SCHEMA      utf8mb4 utf8mb4_bin     NULL
def     infoschema      utf8mb4 utf8mb4_bin     NULL
def     METRICS_SCHEMA  utf8mb4 utf8mb4_bin     NULL
def     mysql   utf8mb4 utf8mb4_bin     NULL
def     PERFORMANCE_SCHEMA      utf8mb4 utf8mb4_bin     NULL
def     test    utf8mb4 utf8mb4_bin     NULL
DROP DATABASE d;
set session sql_mode='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DI
but got(526):
SELECT * from INFORMATION_SCHEMA.SCHEMATA;
CATALOG_NAME    SCHEMA_NAME     DEFAULT_CHARACTER_SET_NAME      DEFAULT_COLLATION_NAME  SQL_PATH        TIDB_PLACEMENT_POLICY_NAME      TIDB_DIRECT_PLACEMENT
def     d       utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     INFORMATION_SCHEMA      utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     infoschema      utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     METRICS_SCHEMA  utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     mysql   utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     PERFORMANCE_SCHEMA      utf8mb4 utf8mb4_bin     NULL    NULL    NULL
def     test    utf8mb4 utf8mb4_bin     NULL    NULL    NULL


```

**Comment by Defined2014** [comment]
Seems like we added `TIDB_PLACEMENT_POLICY_NAME` and `TIDB_DIRECT_PLACEMENT` in `INFORMATION_SCHEMA.SCHEMATA` without change mysql_test.

**Comment by Defined2014** [comment]
@morgo it's caused by #29780, need we add those two colums in `INFORMATION_SCHEMA.SCHEMATA`? Seems this function is not GA yet.

Anyway, I create a PR try to fix it, [#1336](https://github.com/pingcap/tidb-test/pull/1336) PTAL
